appstudio-controller/controllers/appstudio.redhat.com/sandboxprovisioner_controller.go

```diff
@@ -118,6 +118,7 @@ func (r *SandboxProvisionerReconciler) Reconcile(ctx context.Context, req ctrl.R
 		spaceRequest, err = createSpaceRequestForDTC(ctx, r.Client, &dtc)
 		if err != nil {
 			log.Error(err, "failed to create a new SpaceRequest for the DeploymentTargetClaim")
+			logutil.LogAPIResourceChangeEvent(spaceRequest.Namespace, spaceRequest.Name, spaceRequest, logutil.ResourceCreated, log)
```

Comment:
Yes, right, updated

Reply:
Sorry, this should be included inside the condition where we are checking if the space request is nil. Otherwise, we will log even if the space request exists.

Reply:
Yes, it should be after err if condition